init-tools.cmd

```diff
@@ -60,13 +60,22 @@ if NOT exist "%BUILD_TOOLS_PATH%init-tools.cmd" (
 
 :afterbuildtoolsrestore
 
-echo Initializing BuildTools ...
+echo Initializing BuildTools...
 echo Running: "%BUILD_TOOLS_PATH%init-tools.cmd" "%~dp0" "%DOTNET_CMD%" "%TOOLRUNTIME_DIR%" >> "%INIT_TOOLS_LOG%"
 call "%BUILD_TOOLS_PATH%init-tools.cmd" "%~dp0" "%DOTNET_CMD%" "%TOOLRUNTIME_DIR%" >> "%INIT_TOOLS_LOG%"
 set INIT_TOOLS_ERRORLEVEL=%ERRORLEVEL%
 if not [%INIT_TOOLS_ERRORLEVEL%]==[0] (
-	echo ERROR: An error occured when trying to initialize the tools. Please check '%INIT_TOOLS_LOG%' for more details. 1>&2
-	exit /b %INIT_TOOLS_ERRORLEVEL%
+  echo ERROR: An error occured when trying to initialize the tools. Please check '%INIT_TOOLS_LOG%' for more details. 1>&2
+  exit /b %INIT_TOOLS_ERRORLEVEL%
+)
+
+:: Need updated nuget binaries in our cli drop
+echo Updating CLI Nuget Frameworks map...
+robocopy "%TOOLRUNTIME_DIR%" "%TOOLRUNTIME_DIR%\dotnetcli\sdk\%DOTNET_VERSION%" Nuget.Frameworks.dll /XO >> "%INIT_TOOLS_LOG%"
```

Comment:
I was expecting Nuget.*.dll to work as well. What errors are you seeing?

---
In reply to: [78878113](https://github.com/dotnet/corefx/pull/11689#discussion_r78878113) [](ancestors = 78878113)